core-bundle/contao/templates/twig/component/_rich_text.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+    This component outputs rich text content, that was typically composed by
+    an editor in the backend using the tinyMCE editor.
+
+      <div class="rte">
+        <p>Lorem ipsum</p>
+      </div>
+
+    Mandatory variables:
+        @var string text
+
+    Optional variables:
+        @var \Contao\CoreBundle\String\HtmlAttributes attributes
+
+    Note:
+        You can either group all variables in an object named "rich_text" or
+        all directly in the context. Grouping under an object has precedence.
+#}
+
+{% trans_default_domain "contao_default" %}
+
```

Comment:
```suggestion
```

This does not seem to be needed here?

Reply:
You're absolutely right. Copy and paste error. :slightly_smiling_face: